lib/algora_web/controllers/user_auth.ex

```diff
@@ -54,10 +54,14 @@ defmodule AlgoraWeb.UserAuth do
   end
 
   defp redirect_require_login(socket) do
-    socket
-    |> LiveView.put_flash(:error, "Please sign in")
-    |> LiveView.redirect(to: ~p"/auth/login")
-  end
+  
+  case LiveView.get_connect_info(socket).conn
```

Comment:
@zcesur can you please assist on how we can get conn here or what is the correct way of getting the conn here to use in  maybe_store_return_to()